com.musala.simple.students.spring/src/main/java/com/musala/simple/students/spring/web/EventsController.java

```diff
@@ -0,0 +1,22 @@
+package com.musala.simple.students.spring.web;
+
+import java.util.List;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.musala.simple.students.spring.web.dbevents.Event;
+
+@RestController
+public class EventsController {
+
+    @GetMapping(value = "/events")
+    @CrossOrigin(origins = "http://localhost:4200")
```

Comment:
Can you extract this to a constant or, if it is to be changed, a configuration.properties file? :)